.circleci/config.yml

```diff
@@ -241,15 +241,17 @@ jobs:
       - run:
           name: Install protobuf
           command: make proto-tools
-
+      - run:
+          name: "Protobuf Format"
+          command: make proto-format
```

Comment:
By doing the format first, we can rely upon the `git diff` below to catch formatting drift free of charge.